tests/integration/targets/vmware_ops_upload_iso_to_datastore_test/vars/main.yml

```
@@ -0,0 +1,10 @@
+---
+run_on_simulator: false
+
+upload_iso_to_datastore_datacenter: "{{ vcenter_datacenter }}"
+upload_iso_to_datastore_datastore: "{{ shared_storage_01 }}"
+upload_iso_to_datastore_src:  "/tmp/{{ tiny_prefix }}-test.iso"
+upload_iso_to_datastore_dst_folder: "/upload_iso_to_datastore"
```

Confusing naming: it looks like role var but it is not.
```suggestion
folder_name: "/upload_iso_to_datastore"
```